addons/escoria-core/plugin.gd

```diff
@@ -301,6 +301,14 @@ func set_escoria_debug_settings():
 			"hint": PROPERTY_HINT_DIR
 		}
 	)
+	
+	register_setting(
```

Comment:
Does this need to be here? Wasn't it already merged into `dev`?

Reply:
Right this was duplicated for some reason.